packages/lesswrong/lib/collections/automatedContentEvaluations/newSchema.ts

```diff
@@ -34,12 +34,44 @@ const schema = {
       typescriptType: "{ sentence: string; score: number; }[]",
     },
     graphql: {
-      outputType: "[SentenceScore]",
-      canRead: ["admins"],
+      outputType: "[SentenceScore]!",
+      canRead: ["guests"],
     },
+  },
 
+  aiChoice: {
+    database: {
+      type: "VARCHAR(255)",
```

Comment:
I don't think we use varchar(255) anywhere else?  (I agree that `text` is somewhat gratuitous but would rather make the choice to add another DB type consciously rather than sleepwalk into it.  I'm not even sure this will work with how our sql generation works.)

Reply:
I could do `varchar(27)` like we do above?